From 8623e7791172ce80cca3d3793429ced7ee674b8d Mon Sep 17 00:00:00 2001
From: Michel Thierry <michel.thierry@intel.com>
Date: Wed, 26 Apr 2017 12:37:25 -0700
Subject: [PATCH 1550/2302] REVERTME [VPG]: drm/i915/guc: Turn guc submission
 on by default

The powers that be decided that GuC submission is the _future_.

REVERMTE: GuC submission will be enabled by default in upstream
someday, and that patch would arrive from upstream.

Change-Id: Ibad082d1b060abf252c426f152a97609eb0aa5d3
Signed-off-by: Michel Thierry <michel.thierry@intel.com>
---
 drivers/gpu/drm/i915/i915_params.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_params.c b/drivers/gpu/drm/i915/i915_params.c
index 12f995e..4180a34 100644
--- a/drivers/gpu/drm/i915/i915_params.c
+++ b/drivers/gpu/drm/i915/i915_params.c
@@ -63,8 +63,8 @@ struct i915_params i915_modparams __read_mostly = {
 	.verbose_state_checks = 1,
 	.nuclear_pageflip = 0,
 	.edp_vswing = 0,
-	.enable_guc_loading = 0,
-	.enable_guc_submission = 0,
+	.enable_guc_loading = 1,
+	.enable_guc_submission = 1,
 	.guc_log_level = -1,
 	.guc_firmware_path = NULL,
 	.huc_firmware_path = NULL,
@@ -215,11 +215,11 @@ i915_param_named_unsafe(edp_vswing, int, 0400,
 
 i915_param_named_unsafe(enable_guc_loading, int, 0400,
 	"Enable GuC firmware loading "
-	"(-1=auto, 0=never [default], 1=if available, 2=required)");
+	"(-1=auto, 0=never, 1=if available [default], 2=required)");
 
 i915_param_named_unsafe(enable_guc_submission, int, 0400,
 	"Enable GuC submission "
-	"(-1=auto, 0=never [default], 1=if available, 2=required)");
+	"(-1=auto, 0=never, 1=if available [default], 2=required)");
 
 i915_param_named(guc_log_level, int, 0400,
 	"GuC firmware logging level (-1:disabled (default), 0-3:enabled)");
-- 
2.7.4

